screens/game_screen.py

```diff
@@ -51,9 +51,19 @@ def update_buttons(self, response_list):
 
         self.button_columns._set_contents(final_buttons)
 
+    def keypress(self, size, key):
+        """Handle q for quitting and Keypress to get to Help Screen"""
+        key = super().keypress(size, key)
+        if str(key).lower() == 'r':
```

Comment:
can you remove the `r` binding? its just testing code

Reply:
Complete